### PR TITLE
MIR-564 Auto deploy test results to github repository

### PR DESCRIPTION
**Pull request by AMill7**
please not merge yet, need travis tests

### CODE REVIEW

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on $TRAVIS_COMMIT! 
 Plese Check https://github.com/MyCoRe-Travis/test_artifacts/tree/$1

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on d0f0d351ae411fb5c868d436acf227fd11380bf3! 
 Plese Check https://github.com/MyCoRe-Travis/test_artifacts/tree/285-pull_request-50

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 446852ee27fa9dbf9cb03f897d5a7a0fad5a296a! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/288-pull_request-50

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 79f515a1a653065d4df3d50bd120dbca61ee22ae! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/290-pull_request-50

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 54f6bc24499d392bd3affc7b56a8bffa7eb5e3aa! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/340-pull_request-50

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 6c5c5fdeb8081c1ab058d2d20c57f2667887ad16! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/344-pull_request-50

File: travis/autodeploy.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+SAVE=20
+
+FIRST=$(( TRAVIS_BUILD_NUMBER - SAVE ))
+PROTECT=$(seq $FIRST $TRAVIS_BUILD_NUMBER)
+
+eval "$(git for-each-ref --shell --format='git push origin --delete %(refname)' refs/remotes/origin|grep -v $(echo "$PROTECT" |sed -e 's|\(.*\)|refs/remotes/origin/\1|g'|xargs -I repl echo -n repl"\\|" && echo -n 'refs/remotes/origin/HEAD\|refs/remotes/origin/master')|sed -e 's|refs/remotes/origin/||')"
+
+if [ "$TRAVIS_PULL_REQUEST" != "false" ]
+then 
+	curl -H "Authorization: token $GITHUB_TOKEN" -X POST -d "{\"body\":\"Something gone wrong with $TRAVIS_COMMIT! \n Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/$1\"}" https://api.github.com/repos/MyCoRe-Org/mir/issues/$TRAVIS_PULL_REQUEST/comments
```
Comment (yagee-de) [inline travis/autodeploy.sh]: Doesn't `https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/` depend on `$2`?

Comment (AMill7) [inline travis/autodeploy.sh]: $1 is the branch name, so it should be right. BUT somehow it doesn't create a branch if i use this if, but it writes the comments. :(

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 1405bf9d5958816e612bf82eb7c2f9c118eed9da! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/346-pull_request-50

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 41b563a429fb454d2deacfec5d3592e037cb0451! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/348-pull_request-50

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 3d53149ef2d83149c9df0b6b9ccfe792ecdcbd1f! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/350-pull_request-50

File: travis/autodeploy.sh (hunk 1 above)
Comment (yagee-de) [inline travis/autodeploy.sh]: I mean line 29. There is a hard coded string that seems to depend on parameter $2.

Comment (AMill7) [inline travis/autodeploy.sh]: No. $2 is the clone URL. So it only used to clone the repository. 
`https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/` need after that the branch ID.
Like https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/350-pull_request-50.

**Pull request merged**